metadata-ingestion/tests/unit/test_classification.py

```diff
@@ -157,3 +157,74 @@ def test_incorrect_custom_info_type_config():
                 },
             }
         )
+
+
+def test_exclude_name_config():
+    config = DataHubClassifier.create(
+        config_dict={
+            "confidence_level_threshold": 0.7,
+            "info_types_config": {
+                "Email_Address": {
+                    "Prediction_Factors_and_Weights": {
+                        "Name": 1,
+                        "Description": 0,
+                        "Datatype": 0,
+                        "Values": 0,
+                    },
+                    "ExcludeName": ["email_sent", "email_received"],
+                    "Name": {
+                        "regex": [
+                            "^.*mail.*id.*$",
+                            "^.*id.*mail.*$",
+                            "^.*mail.*add.*$",
+                            "^.*add.*mail.*$",
+                            "email",
+                            "mail",
+                        ]
+                    },
+                    "Description": {"regex": []},
+                    "Datatype": {"type": ["str"]},
+                    "Values": {"prediction_type": "regex", "regex": [], "library": []},
+                }
+            },
+        }
+    ).config
+    if config.info_types_config["Email_Address"].ExcludeName is not None:
```

Comment:
can remove this if/else by asserting presence of ExcludeName directly.
```suggestion
    assert config.info_types_config["Email_Address"].ExcludeName is not None
```